spec/views/groups/show.html.haml_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "groups/show", type: :view do
+  before(:each) do
+    @group = assign(:group, Group.create!(
+      :category => "Category",
+      :boolean => ""
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.